package.json

```diff
@@ -1,8 +1,30 @@
 {
+  "name": "speedometer",
+  "version": "3.0.0",
```

Comment:
I know we aren't actually publishing this, but perhaps it makes sense to still mark as alpha or some other way to indicate that it's not complete?

Reply:
I can change it to "3.0.0-alpha", or remove the key/value.